Scripts/DCS-BIOS/doc/userguide.adoc

```diff
@@ -139,7 +139,7 @@ dofile(lfs.writedir()..[[Scripts\DCS-BIOS\BIOS.lua]])
 WARNING: You should never use any word processing software to edit text files.
 Word processing software saves formatting information in the file which will render it unusable.
 
-Note: If you use the Raygun Software from Buddyfox (A-10C UFC), his entry must stand above the DCS-BIOS entry in the Export.lua !!!
+Note: If you use the Raygun Software from Buddyfox (A-10C UFC), this entry must be *before* the DCS-BIOS entry in the Export.lua !

```

Comment:
Since we removed all the globals in bios, I wonder if this is still true...

Reply:
Could be yeah. Let's get back to that later.